ui/ducks/send/send.js

```diff
@@ -1746,6 +1824,12 @@ export function signTransaction() {
           eip1559support ? eip1559OnlyTxParamsToUpdate : txParams,
         ),
       };
+      await dispatch(
+        addHistoryEntry(
+          `sendFlow - user clicked next and transaction should be updated in controller`,
+        ),
+      );
+      await dispatch(updateTransactionSendFlowHistory(id, state[name].history));
```

Comment:
Can this rather be an implicit part of signTransaction `updateTransactionSendFlowHistory` so that we do not need to call it from UI.

Reply:
i'm unclear as to what you mean. If its a new transaction we send it with the transaction params to create the new transaction. If its an existing one (edited) then we call this method.

Reply:
I mean can this be part of other action at the backend may be `addUnapprovedTransactionAndRouteToConfirmationPage`

Reply:
that already is the case for that flow, but we also need the update flow for an edit transaction, and its possible to encapsulate the different update methods into a single action but I was following the pattern in place (updateEditableParams/updateTransactionGasFees)